src/app/[locale]/projects/detail/[id]/components/ProjectDetailTab.tsx

```diff
@@ -0,0 +1,269 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+"use client"
+
+import { Col, Row, ListGroup, Tab, Button, Dropdown } from 'react-bootstrap'
+import Summary from './Summary'
+import React, { useState, useEffect } from 'react'
+import PageSpinner from '@/components/Spinner/Spinner'
+
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+
+import ApiUtils from "@/utils/api/api.util"
+import CommonUtils from '@/utils/common.utils'
+import { Session } from '@/object-types/Session'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { signOut } from 'next-auth/react'
+
+import { SummaryDataType } from '@/object-types/SummaryDataType'
+import { notFound } from 'next/navigation'
+
+export default function ViewProjects({ session, projectId }: { session: Session, projectId: string }) {
+
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [loading, setLoading] = useState(true)
+    const [summaryData, setSummaryData] = useState<SummaryDataType>(
+        {
+            description: "",
+
+            // General Information
+            id: "", 
+            name: "", 
+            version: "",
+            visibility: "", 
+            createdOn: "", 
+            createdBy: { name: "", email: ""},
+            modifiedOn: "", 
+            modifiedBy: { name: "", email: ""},
+            projectType: "",
+            domain: "",
+            tag: "",
+            externalIds: new Map<string, string>,
+            additionalData: new Map<string, string>,
+            externalUrls: new Map<string, string>,
+        
+            // Roles
+            group: "",
+            projectResponsible: { name: "", email: ""},
+            projectOwner: { name: "", email: ""},
+            ownerAccountingUnit: "",
+            ownerBillingGroup: "",
+            ownerCountry: "",
+            leadArchitect: { name: "", email: ""},
+            moderators: [],
+            contributors: [],
+            securityResponsibles: [],
+            additionalRoles: new Map<string, string>,
+
+            // Project Vendor
+            vendorFullName: "",
+            vendorShortName: "",
+            vendorUrl: ""
+        }
+    )
+
+    useEffect(() => {
+        const controller = new AbortController()
+        const signal = controller.signal
+
+        ;(async () => {
```

Comment:
To much requests to fetch project summary data. Need to re-define API endpoint to fetch the necessary data simultaneously

Reply:
Yes, too many requests. The API endpoint will be changed to reduce the number of requests.